core/src/cats/bvfs.cc

```diff
@@ -144,9 +138,7 @@ void BareosDb::BuildPathHierarchy(JobControlRecord* jcr,
 
       if (!QUERY_DB(jcr, cmd)) { goto bail_out; /* Query failed, just leave */ }
 
-      /*
-       * Do we have a result ?
-       */
+      // Do we have a result ?
```

Comment:
Thanks for pointing that one out. I probably couldn't have figured it out by looking at `SqlNumRows() > 0`
</sarcasm>